module/VuFind/src/VuFind/DigitalContent/FakeOverdriveConnector.php

```diff
@@ -313,7 +313,7 @@ public function getAuthHeader()
     }
 
     /**
-     * Returns permanant links for Ovedrive resources
+     * Returns permanent links for Overdrive resources
```

Comment:
This should actually be `OverDrive` rather than `Ovedrive` -- it was a double typo!

Reply:
All variants seem to occur in the code (and in filenames like Overdrive.ini).

Reply:
Yes, I believe the service rebranded (or at very least there was an initial misunderstanding about preferred capitalization). We unfortunately can't fix the filename issues without causing disruption, but I'm at least trying to be consistent in user-facing messages and comments.